libraries/chain/chain_controller.cpp

```diff
@@ -210,7 +211,26 @@ bool chain_controller::_push_block(const signed_block& new_block)
 
    try {
       auto session = _db.start_undo_session(true);
+      auto exec_start = std::chrono::high_resolution_clock::now();
       apply_block(new_block, skip);
+      if( (fc::time_point::now() - new_block.timestamp) < fc::seconds(60) )
+      {
+         auto exec_stop = std::chrono::high_resolution_clock::now();
+         auto exec_ms = std::chrono::duration_cast<std::chrono::milliseconds>(exec_stop - exec_start);      
+         size_t trxcount = 0;
+         for (const auto& cycle : new_block.cycles)
+   	      for (const auto& thread : cycle)
+   		      for (const auto& trx : thread.user_input) 
+                  trxcount++;
```

Comment:
Bad indentation on above lines... Just replace the innermost loop with `trxcount += thread.user_input.size()` -- might as well skip a few iterations, ya?

Reply:
Will do